WordPress/src/main/java/org/wordpress/android/ui/stockmedia/StockMediaRetainedFragment.java

```diff
@@ -0,0 +1,43 @@
+package org.wordpress.android.ui.stockmedia;
+
+import android.app.Fragment;
```

Comment:
Could we use the support version of the `Fragment` instead? That'll require a few changes in the Activity, but I'll not mention them separately.

I really think we should pick one of the Fragment versions and use it consistently. I've been trying to make that happen, but have been doing a really poor job so far. I probably need to fix all of it in a PR and then write a P2 post.